src/components/Profile/ProfileImage/index.js

```diff
@@ -0,0 +1 @@
+export {ProfileImage} from "./ProfileImage";
```

Comment:
Следите за кавычками
```suggestion
export {ProfileImage} from './ProfileImage';
```

Reply:
Сделано